desktop/src/com/mygdx/game/client/ui/NotPlayerTurnDialogFactory.java

```diff
@@ -0,0 +1,39 @@
+package com.mygdx.game.client.ui;
+
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.Dialog;
+import com.mygdx.game.assets.GameScreenAssetPaths;
+import com.mygdx.game.assets.GameScreenAssets;
+import com.mygdx.game.client.di.StageModule;
+import lombok.NonNull;
+import lombok.extern.java.Log;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+@Log
+@Singleton
+public class NotPlayerTurnDialogFactory {
+
+  private final GameScreenAssets assets;
+  private final Stage stage;
+
+  @Inject
+  public NotPlayerTurnDialogFactory(
+      @NonNull GameScreenAssets assets,
+      @NonNull @Named(StageModule.GAME_SCREEN) Stage stage
+  ) {
+    this.assets = assets;
+    this.stage = stage;
+  }
+
+  public void createNotPlayerTurnDialog() {
```

Comment:
zmień na createAndShow, bo taki już się chyba tworzył schemat

Reply:
Okej ale trochę mi się niie podoba, teraz klasa mówi do czego to służy ale jeśli w jednej klasie będzie kilka dialogów to może być mylące